helm/prometheus-rules/templates/alerting-rules/app.rules.yml

```diff
@@ -140,6 +141,40 @@ spec:
         sig: none
         team: honeybadger
         topic: releng
+    # Alert Atlas about EFK deployment failed as detected by app platform.
```

Comment:
Is it not possible to use the team annotation set in the chart instead ? cc @JosephSalisbury

Reply:
Routing by app name will work for a few apps but yes we could use the team label instead.

The [WorkloadClusterAppFailed](https://github.com/giantswarm/prometheus-rules/blob/4e7df4887b8b36a9f8af436c6bf5a4fcf37bbe55/helm/prometheus-rules/templates/alerting-rules/app.rules.yml#L127-L142) alert needs to be split by team like we do for MCs. e.g. [ManagementClusterAppFailedAtlas](https://github.com/giantswarm/prometheus-rules/blob/4e7df4887b8b36a9f8af436c6bf5a4fcf37bbe55/helm/prometheus-rules/templates/alerting-rules/app.rules.yml#L14-L29)

We just need to be careful with the statuses. I would just route `failed` apps for now. With other statuses like `not-installed` if chart-operator is broken we could page multiple teams which would be dumb